.gitignore

```diff
@@ -2,9 +2,5 @@
 # will have compiled files and executables
 /target/
 
-# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
-# More information here https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html
-Cargo.lock
-
```

Comment:
I'm leaning towards wanting to leave this in, and just use the full "x.y.x" versions for dependencies, to trigger dependabot.

Reply:
:facepalm: I read that backwards, i'll add back